pom.xml

```diff
@@ -131,9 +130,8 @@
         </profile>
 
         <profile>
-            <id>all-other</id> <!-- used to update project versions and check all modules compilation -->
-            <modules> <!-- sorted alphabetically -->
-                <module>modules/web-console</module>
+            <id>all-other</id> <!-- Used to update project versions and check all modules compilation. -->
```

Comment:
How about to remove this profile completely if no modules left there?

Reply:
@vveider From my opinion, this is out of this issue scope.

Reply:
But empty profile is a bug, leftover.
@nva, WDYT?

Reply:
@vveider I removed this profile and its usages (DEVNOTES.txt, update-versions.sh, and build.sh in ggprivate repository).